### PR TITLE
add connector description

### PR DESCRIPTION
**Pull request by Nicole00**

### CODE REVIEW

**Comment by Nicole00** [comment]
This PR adds English description for spark-connector, @Amber1990Zhang @jamieliu1023  please help me review this, thanks

File: nebula-spark-connector/README.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
+1. Install Nebula Java Client 2.0.
 
     ```bash
     $ git clone https://github.com/vesoft-inc/nebula-java.git
     $ cd nebula-java
     $ mvn clean install -Dmaven.test.skip=true -Dgpg.skip -Dmaven.javadoc.skip=true  
     ```
 
-    打包结束后，在本地 Maven Repository 仓库中可以看到生成的 /com/vesoft/client/2.0.0-rc1/client-2.0.0-rc1.jar。
+    After the installing, you can see the newly generated /com/vesoft/client/2.0.0-rc1/client-2.0.0-rc1.jar in your local Maven repository.
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:20]: installation

File: nebula-spark-connector/README.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
+    After the packaging, you can see the newly generated nebula-spark-connector-2.0.0.jar under the nebula-spark-utils/nebula-spark-connector/target/ directory.
 
-## 特性
-* 提供了更多连接配置项，如超时时间、连接重试次数、执行重试次数
-* 提供了更多数据配置项，如写入数据时是否将 vertexId 同时作为属性写入、是否将 srcId、dstId、rank 等同时作为属性写入
-* Spark Reader 支持无属性读取，支持全属性读取
-* Spark Reader 支持将 Nebula Graph 数据读取成 Graphx 的 VertexRD 和 EdgeRDD，支持非 Long 型 vertexId
-* Nebula Spark Connector 2.0 统一了 SparkSQL 的扩展数据源，统一采用 DataSourceV2 进行 Nebula Graph 数据扩展
+## New Features 
+* Supports more connection configurations, such as timeout, connectionRetry, executionRetry
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:33]: connectionRetry, and executionRetry

File: nebula-spark-connector/README.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
+* Supports more data configurations, such as if vertexId also as property, if srcId, dstId and rank also as properties
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:34]: Add a period for each sentence.

File: nebula-spark-connector/README.md [hunk 4]
```diff
@@ -1,50 +1,50 @@
...
+* Spark Reader Supports non-property reading, all-property reading, specific-properties reading
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:35]: ditto

File: nebula-spark-connector/README.md [hunk 5]
```diff
@@ -1,50 +1,50 @@
...
+* Spark Reader Supports reading Nebula Graph's data to Graphx's VertexRD and EdgeRDD, and supports non-Long type vertexId
+* Nebula Spark Connector 2.0 uniformly uses SparkSQL's DataSourceV2 for data source expansion.
 
-## 使用说明
+## How to Use
 
-  将 DataFrame 作为点写入 Nebula Graph :
+  Write DataFrame into Nebula Graph as Vertex:
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:41]: as vertices

File: nebula-spark-connector/README.md [hunk 6]
```diff
@@ -74,11 +73,44 @@ Nebula Spark Connector 2.0 依赖 Nebula Java Client 2.0。
...
+      .withPartitionNum(10)
+      .build()
+
+    val vertex = spark.read.nebula(config, nebulaReadVertexConfig).loadVerticesToGraphx()
+    val edgeRDD = spark.read.nebula(config, nebulaReadEdgeConfig).loadEdgesToGraphx()
+    val graph = Graph(vertexRDD, edgeRDD)
+  ```
+  After getting Graphx's Graph, you can develop graph algorithms in Graphx like [Nebula-Spark-Algorithm](https://github.com/vesoft-inc/nebula-java/tree/v1.0/tools/nebula-algorithm).
+
+For more use details please refer to [Example](https://github.com/vesoft-inc/nebula-spark-utils/tree/master/example/src/main/scala/com/vesoft/nebula/examples/connector).
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:108]: For more information on usage, refer ...

File: nebula-spark-connector/README.md [hunk 7]
```diff
@@ -74,11 +73,44 @@ Nebula Spark Connector 2.0 依赖 Nebula Java Client 2.0。
...
+
+## How to Contribute
+
+Nebula Spark Connector 2.0 is a completely opensource project, opensource enthusiasts are welcome to participate in the following ways:
 
-- 前往 [Nebula Graph 论坛](https://discuss.nebula-graph.com.cn/ "点击前往“Nebula Graph 论坛") 上参与 Issue 讨论，如答疑、提供想法或者报告无法解决的问题
-- 撰写或改进文档
-- 提交优化代码
+- Go to [Nebula Graph Forum](https://discuss.nebula-graph.com.cn/ "go to“Nebula Graph Forum") to participate in issue discussions, such as answering questions, providing ideas, or reporting unsolvable problems;
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:114]: Go to [Nebula Graph Forum](https://discuss.nebula-graph.com.cn/ "go to“Nebula Graph Forum") for help.

File: nebula-spark-connector/README.md [hunk 8]
```diff
@@ -74,11 +73,44 @@ Nebula Spark Connector 2.0 依赖 Nebula Java Client 2.0。
...
+- write or improve documents;
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:115]: Write

File: nebula-spark-connector/README.md [hunk 9]
```diff
@@ -74,11 +73,44 @@ Nebula Spark Connector 2.0 依赖 Nebula Java Client 2.0。
...
+- submit code to add new features or fix bugs
```
Comment (Amber1990Zhang) [inline nebula-spark-connector/README.md:116]: ditto

File: nebula-spark-connector/README.md (hunk 5 above)
Comment (Nicole00) [reply nebula-spark-connector/README.md]: good catch, thanks~

File: nebula-spark-connector/README.md (hunk 7 above)
Comment (Nicole00) [reply nebula-spark-connector/README.md]: Maybe users can also answer questions and propose new requirements.

Comment (Amber1990Zhang) [reply nebula-spark-connector/README.md]: Go to Nebula Graph Forum to discuss with other users. You can raise your own questions, help others, share your thoughts.

Comment (Nicole00) [reply nebula-spark-connector/README.md]: got it! Thanks

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**